files/gsoc-bootstrap/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2.1'
+
+services:
+  openmrs-referenceapplication-mysql:
+    restart: "always"
+    image: mysql:5.6
+    command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci"
+    environment:
+      MYSQL_DATABASE: ${MYSQL_DB:-openmrs}
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-Admin123}
+      MYSQL_USER: ${MYSQL_USER:-openmrs}
+      MYSQL_PASSWORD: ${MYSQL_PASSWORD:-Admin123}
+    healthcheck:
+      test: "exit 0"
+    volumes:
+      - ./dbdump:/docker-entrypoint-initdb.d
+      # - openmrs-referenceapplication-mysql-data:/var/lib/mysql  # uncomment to keep database between docker runs
+
+  openmrs-referenceapplication:
+    restart: "always"
+    image: openmrs/openmrs-reference-application-distro:${OPENMRS_VERSION:-gsoc-bootstrap}
+    depends_on:
+      - openmrs-referenceapplication-mysql
+    ports:
+      - "8083:8080"
+    environment:
+      DB_DATABASE: ${MYSQL_DB:-openmrs}
+      DB_HOST: openmrs-referenceapplication-mysql
+      DB_USERNAME: ${MYSQL_USER:-openmrs}
+      DB_PASSWORD: ${MYSQL_PASSWORD:-Admin123}
+      DB_CREATE_TABLES: 'false'
+      DB_AUTO_UPDATE: 'false'
+      MODULE_WEB_ADMIN: 'false'
```

Comment:
@Akayeshmantha set this to true so that you can upload modules

Reply:
Sure will do.